pkgs/development/python-modules/imageio-ffmpeg/default.nix

```diff
@@ -1,29 +1,38 @@
 { lib
 , buildPythonPackage
-, fetchPypi
-, fetchpatch
 , isPy3k
+, fetchPypi
+, substituteAll
+, ffmpeg
+, python
 }:
 
 buildPythonPackage rec {
   pname = "imageio-ffmpeg";
-  version = "0.4.2";
+  version = "0.4.3";
+
+  disabled = !isPy3k;
 
   src = fetchPypi {
     inherit pname version;
-    sha256 = "13b05b17a941a9f4a90b16910b1ffac159448cff051a153da8ba4b4343ffa195";
+    sha256 = "f826260a3207b872f1a4ba87ec0c8e02c00afba4fd03348a59049bdd8215841e";
   };
-  patches = [ (fetchpatch {
-    # Fixes compatibility with python3.9
-    # Should be included in the next release after 0.4.2
-    url = "https://github.com/imageio/imageio-ffmpeg/pull/43/commits/b90c39fe3d29418d67d953588ed9fdf4d848f811.patch";
-    sha256 = "0d9kf4w6ldwag3s2dr9zjin6wrj66fnl4fn8379ci4q4qfsqgx3f";
-  })];
 
-  disabled = !isPy3k;
+  patches = [
+    (substituteAll {
+      src = ./ffmpeg-path.patch;
+      ffmpeg = "${ffmpeg}/bin/ffmpeg";
+    })
+  ];
+
+  checkPhase = ''
```

Comment:
Should probably run the preCheck and postCheck hooks.

Reply:
I never got why, but sure.